utils/xz/Makefile

```diff
@@ -9,12 +9,11 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=xz
 PKG_VERSION:=5.2.2
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=http://tukaani.org/xz \
-		http://fossies.org/linux/misc
-PKG_MD5SUM:=f90c9a0c8b259aee2234c4e0d7fd70af
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=@SF/lzmautils
+PKG_MD5SUM:=e26772b69940085c0632589ab1d52e64
```

Comment:
I think we should at least keep the tukaani.org URL as it seems (one of) the official site of the package besides the fact that I encountered many times the situation where connections to sourceforge just timed out.

BTW., please consider be verbose about the change in commit message as you have presented in the comment.  It may help facilitate the review process.